pkg/composer/serviceparser/build.go

```diff
@@ -25,9 +25,9 @@ import (
 	"github.com/compose-spec/compose-go/v2/types"
 	securejoin "github.com/cyphar/filepath-securejoin"
 
-	"github.com/containerd/containerd/v2/pkg/identifiers"
 	"github.com/containerd/errdefs"
 	"github.com/containerd/log"
+	"github.com/containerd/nerdctl/v2/pkg/identifiers"
```

Comment:
@Shubhranshu153 can you rebase? We just introduced some linting on imports.
Once rebased, you should be able to autotidy with `make lint-fix-imports`

Reply:
Also `s/remove/relax/` in the commit message

Reply:
sure let me try that out

Reply:
updated it, the test seems to fail windows in setup can we retrigger.

Reply:
Just ignore canary windows failures for now (#3306). There is a bug in the go setup action that prevents it from functioning with full go version ending in .0.

Reply:
@Shubhranshu153 you may rebase to get CI failure fixed.

Reply:
rebased